pkg/apis/noobaa/v1alpha1/noobaaaccount_types.go

```diff
@@ -69,7 +69,7 @@ type NooBaaAccountSpec struct {
 
 	// ForceMd5Etag specifies whether MD5 Etags should be calculated for the account or not
 	// +optional
-	ForceMd5Etag *bool `json:"force_md5_etag,omitempty"`
+	ForceMd5Etag bool `json:"force_md5_etag,omitempty"`
```

Comment:
Why is this change necessary? 
With this code change, the user must set a value for `ForceMd5Etag`. If the user leaves this field empty, it will implicitly be set to `false` and override any other setting\default in noobaa-core.

Reply:
Done the required changes as per discussed in slack, everything working as expected. Thanks